### PR TITLE
DOPS-4261 Enabled bulk insert and improved exception debugging

### PR DESCRIPTION
**Pull request by gecbla**
@MiroslavArbor You can add this version of SDK to DI project in following ways:

1. run following commands locally or form Docker container (recomended way)
`composer remove arbor-education/arbor-sdk-php`
`composer require arbor-education/arbor-sdk-php:dev-improvement/DOPS-4261-enabled-bulk-insert-and-improved-exception-debugging`

2. by using symlink
- clone SDK project to your computer
- checkout branch `improvement/DOPS-4261-enabled-bulk-insert-and-improved-exception-debugging`
- delete folder `vendor/arbor-education/arbor-sdk-php` in your DI project
- symlink location of SDK project source code with folder of SDK package in vendor directory of DI project by using relative path (due to using Docker containers)
- `ln -s ./relative-path/to/sis-sdk-folder ./relative-path/to/di-folder/vendor/arbor-education/arbor-sdk-php`

### CODE REVIEW

**Comment by gecbla** [comment]
Looks like it's high time for this PR to come out of shadows and see some day light 😀 

DI team continued work from the point where Dutek left with ticket DOPS-4261 and changes made on [SIS](https://github.com/arbor-education/sis/pull/16782) and [SDK](https://github.com/arbor-education/sis-sdk-php/pull/31/files) side. We also have some basic banckmarks which can be found [here](https://orchard.atlassian.net/browse/IT-10185).

@AnthonyPorthouse @aaronlinley @ronappleton  As project currently doesn't have CODEOWNER file I'm tagging you all so I can grab you attention and get some approvals. If you have any suggestion(s) and need some clarification on the work we done, we can setup a meeting and go over all those things.

File: examples/student-create.php
Comment (aaronlinley) [inline examples/student-create.php]: I think this should be
```suggestion
    printf('Failed to insert person: %s%s', $result['status']['errors'][0], PHP_EOL);
```

**Review by aaronlinley — COMMENTED** [review]
Looks mostly ok to me, just one typo I spotted

File: examples/student-create.php
Comment (ronappleton) [reply examples/student-create.php]: yup seconded

Comment (gecbla) [reply examples/student-create.php]: @AnthonyPorthouse Typo in debug message is fixed, so can someone approve this PR please?

**Review by AnthonyPorthouse — APPROVED** [review]

**Pull request merged**
